action.yml

```diff
@@ -83,11 +83,11 @@ inputs:
   commitUserName:
     description: Name used for the commit user
     required: false
-    default: GitHub Actions
+    default: 'github-actions[bot]'
   commitUserEmail:
     description: Email address used for the commit user
     required: false
-    default: actions@github.com
+    default: '41898282+github-actions[bot]@users.noreply.github.com'
```

Comment:
Hey, thanks for your contribution. Is this rnd number recommended / a general best practice?

Reply:
The ID number is part of noreply address for users created after a certain date (the documentation on it is [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address)). If you look at the [API](https://api.github.com/users/github-actions%5Bbot%5D), the user was created on 2018-07-30T09:30:16Z, a year after they started using their new format. You can also see the ID there. When you just simply make a commit in a workflow, this is the email and username that is being used by Github.

Reply:
Okay, thanks for the explanation